anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/ewma/EwmaAnomalyDetector.java

```diff
@@ -47,11 +47,8 @@
  * @see <a href="https://www.itl.nist.gov/div898/handbook/pmc/section3/pmc324.htm">EWMA Control Charts</a>
  */
 @Data
-public final class EwmaAnomalyDetector implements AnomalyDetector {
-    
-    @NonNull
-    private UUID uuid;
-    
+public final class EwmaAnomalyDetector extends BasicAnomalyDetector<EwmaParams> {
```

Comment:
Similar comments as above regarding detector/params and Law of Demeter. I'll stop mentioning it but take care of any cases.